aries-site/src/pages/components/card.mdx

```diff
@@ -71,32 +134,106 @@ A card by nature has limited real-estate, so it should present summarized inform
 
 This type of card is more commonly used within marketing contexts and should be used sparingly within HPE products and services. Secondary purposes might include an action to navigate to a screen with greater detail on the offering.
 
+#### Anatomy
+
+Use the elements below to craft the appropriate amount of visual emphasis for the card's use case. See the call-to-action card examples for best practices.
+
+DIAGRAM TO BE ADDED
+
+1. **Card identifier**: This region is intended to provide the user with context on what the card is about.
```

Comment:
I'm glad you made this a region with title, pretitle, subtitle as elements making up this region. 

Is a logo, icon, avatar part of this region? Not applicable for CTA cards?

Reply:
I think a logo, icon, avatar could be appropriate for an identifier, but I hadn't seen many use cases of CTA cards with one of those visual elements which is why I left it out for now.